apps/designer/app/designer/features/sidebar-left/panels/pages/settings-panel.tsx

```diff
@@ -0,0 +1,61 @@
+import {
+  styled,
+  keyframes,
+  Collapsible,
+  Box,
+} from "@webstudio-is/design-system";
+
+const CollapsibleRoot = styled(Collapsible.Root, {
+  position: "absolute",
+  left: "100%",
+  top: 0,
+  bottom: 0,
+  display: "flex",
+  flexDirection: "column",
+  marginLeft: 1,
+});
+
+const openKeyframes = keyframes({
+  from: { width: 0 },
+  to: { width: "var(--radix-collapsible-content-width)" },
+});
+
+const closeKeyframes = keyframes({
+  from: { width: "var(--radix-collapsible-content-width)" },
+  to: { width: 0 },
+});
+
+const CollapsibleContent = styled(Collapsible.Content, {
+  overflow: "hidden",
+  flexGrow: "1",
+  display: "flex",
+  flexDirection: "column",
+  '&[data-state="open"]': { animation: `${openKeyframes} 200ms ease-out` },
+  '&[data-state="closed"]': { animation: `${closeKeyframes} 200ms ease-out` },
+});
+
+export const SettingsPanel = ({
```

Comment:
In the end this panel will be needed for more than pages, but we can keep it here for now if you think its risky to make it reusable right away

Reply:
Will do this later, it's kinda WIP yet